client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -495,19 +495,21 @@ export class DomainDetailsForm extends PureComponent {
 		return this.inputRefCallbacks[ name ];
 	}
 
-	renderCurrentForm() {
+	renderCurrentForm( needsOnlyGoogleAppsDetails ) {
```

Comment:
Would it be possible to use a React Component for each address format? i.e. `<USAdressForm>` and `<EuropeanAddressForm>`? That way it'd be easier for designers to work on styling.

Reply:
> That way it'd be easier for designers to work on styling.

That's quite a bit of value right there. Not only that, whenever we require new region-specific formats or changes to existing formats, we can work with independent components. It might help to reduce conditional block chaos later. Thanks!